spinalcordtoolbox/process_seg.py

```diff
@@ -54,15 +75,15 @@ def compute_shape(segmentation, angle_correction=True, param_centerline=None, ve
     if angle_correction:
```

Comment:
Not a problem at all but maybe the else statement could be avoided writing the code like this: 

```python
fit_results = None
if angle_correction:
     # compute the spinal cord centerline based on the spinal cord segmentation
     # here, param_centerline.minmax needs to be False because we need to retrieve the total number of input slices
     _, arr_ctl, arr_ctl_der, fit_results = get_centerline(im_segr, param=param_centerline, verbose=verbose)
```

Not a big change. If you consider it is better to leave it as it is, it's fine too!

Reply:
we can go your way, can you do those changes? thanks!

Reply:
Sure, I'll do it!